openmc/model/triso.py

```diff
@@ -21,6 +21,11 @@
 MAX_PF_CRP = 0.64
```

Comment:
In the spirit of cleanup, these concise variables (related to the packing fraction limits?) probably warrant a comment for clarity...